src/mainboard/clevo/adl-p/acpi/touchpad.asl

```diff
@@ -0,0 +1,16 @@
+External (\_SB.PCI0.I2C0.H038, DeviceObj)
+External (\_SB.PCI0.I2C0.H038._STA, IntObj)
+
+//Scope (PCI0.LPCB.EC0)
```

Comment:
Why are these uncommented?

Reply:
This is no longer the case after rebasing it again on clevo/adl-p-UEFI. But to be clear, I first copied this method from tgl-u to make the build proccess go through since this was the error in ASL, then had to get rid of this scope because otherwise I got an "out of scope" error.